bindings/go/src/fdb/database.go

```diff
@@ -62,7 +62,7 @@ type DatabaseOptions struct {
 
 // Close will close the Database and clean up all resources.
 // You have to ensure that you're not resuing this database.
-func (d *Database) Close() {
+func (d Database) Close() {
```

Comment:
I've not written go for a long time, but I believe this wont break any existing code, because Go you don't have to do anything special to invoke method from a pointer (like we do in C/C++). 

If its not breaking anything, the change looks good to me.

Reply:
It's not breaking anything on Go side and also not on C side, to the best of my understanding. Calling `Close()` again was not supported before this PR and remains unsupported.